base-node/index.d.ts

```diff
@@ -30,7 +30,8 @@ type Message =
   // Inaccurate type until https://github.com/microsoft/TypeScript/issues/26113
   ['sync', number, object, object] |
   ['synced', number] |
-  ['debug', 'error', string]
+  ['debug', 'error', string] |
+  ['headers', object]
 
 /**
  * Abstract interface for connection to synchronize logs over it.
```

Comment:
You forget to add types for `setLocalHeaders` method and `remoteHeaders` property.

Reply:
Completely forget about it. Sorry. I've also added new event `headers` to event listener type.